### PR TITLE
Portfolio mobile version

### PR DESCRIPTION
**Pull request by kidde60**
### Portfolio: mobile version
**On this webpage I:**

- added the work section.
- added background images.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,5 +58,81 @@ <h2>Software Developer</h2>
...
     </section>
+    <section id="work">
+        <div class="work">
+            <img src="images/arrow.svg" alt="arrow">
+            <h2>work</h2>
+        </div>
+        <div class="recent-work">
+
+            <h1>MY RECENT WORK</h1>
+            <hr>
```
Comment (koachcode) [inline index.html]: - Please the horizontal line under the `My Recent Work` should not take the full width of the webpage as it is on your design currently. Also the line color should be `#646369` with a hight of 1px just as in the Figma design

##### Your design 
Screenshot from 2022-06-30 07-53-56

##### Figma design
Screenshot from 2022-06-30 07-58-16

File: index.html [hunk 2]
```diff
@@ -58,5 +58,81 @@ <h2>Software Developer</h2>
...
+                <img src="images/placeholder.png" alt="placeholder">
+            </div>
+            <div class="holder">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="list">
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li class="ruby">Ruby</li>
```
Comment (koachcode) [inline index.html]: - Please use the appropriate font size for the paragraph as it is in the Figma design `inter` 
- Please use the proper color of the divider in between each technology used for the project, yours is currently white, use the Figma color

##### Your design 
Screenshot from 2022-06-30 08-04-04

##### Figma design

Screenshot from 2022-06-30 08-04-25

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @kidde60 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- No linter errors :+1: 
- Good commit messages :+1: 
- Good PR summary :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Your Code reviewer on slack
- `@kanu mike chibundu`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @kidde60, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**